File: ios/NativeComponent/StartScreens/AntenaModel.swift
//
//  AntenaModel.swift
//  STB
//
//  Created by 沈凯 on 2018/5/4.
//  Copyright © 2018年 Ssky. All rights reserved.
//

import UIKit
import STBAPI

enum AntenaArrayType {
    case none
    case single
    case transponder
    case mixture
}

class AntenaModel: NSObject {
//    Title Name
    var title: String!
//    Array Type
    var arrayType: AntenaArrayType!
//    Sigle Array
    var sigleArray: [String]!
//    Transponder Array
    var transponderArray: [DatabaseTransponderModel]!
//    Mixture Array 1
    var mixtureArray1: [String]!
//    Mixture Array 2
    var mixtureArray2: [String]!
//    Index
    var index: Int!
}

File: android/app/src/main/java/com/onapptv/ConnectionView/Custom/Model/AntenaModel.java
package com.onapptv.ConnectionView.Custom.Model;

import com.onapptv.ConnectionView.Custom.View.Antena.AntenaArrayType;

import java.util.List;

import tv.hi_global.stbapi.Model.DatabaseTransponderModel;

public class AntenaModel {
    //    Title Name
    public String title;
    //    Array Type
    public AntenaArrayType arrayType;
    //    Sigle Array
    public String[] sigleArray;
    //    Transponder Array
    public List<DatabaseTransponderModel> transponderArray;
    //    Mixture Array 1
    public String[] mixtureArray1;
    //    Mixture Array 2
    public String[] mixtureArray2;
    //    Index
    public int index;
}
